dashboard/config/initializers/script_preload.rb

```diff
@@ -0,0 +1,2 @@
+# Preload scripts before application fork
+Script.script_cache
```

Comment:
hmm, appears so... I could add a check to only preload the script cache if the db has already initialized. any other ideas on how to workaround this ordering issue?